suite/tests/CMakeLists.txt

```diff
@@ -4170,8 +4170,8 @@ endif (BUILD_CLIENTS)
 # ##########################################################################
 # drpt2trace tests
 # We use the libipt/script/perf-get-opts.bash script to get the options for drpt2trace's tests.
-if(BUILD_CLIENTS)
-  if(HAVE_LIBIPT)
+if (BUILD_CLIENTS)
+  if (BUILD_PT_POST_PROCESSOR)
```

Comment:
BUILD_PT_TRACER AND BUILD_PT_POST_PROCESSOR

Reply:
Done.